base/docs/basedocs.jl

```diff
@@ -2028,6 +2030,62 @@ for bit in (8, 16, 32, 64, 128)
     end
 end
 
+@eval begin
+    """
+        Int$(Sys.WORD_SIZE) <: Signed
+
+    $(Sys.WORD_SIZE)-bit signed integer type.
+    On $(Sys.WORD_SIZE)-bit systems, `Int` is an alias for `Int$(Sys.WORD_SIZE).`
+
+    See also: [`Int`](@ref)
+    """
+    $(Symbol("Int", Sys.WORD_SIZE))
+
+    """
+        UInt$(Sys.WORD_SIZE) <: Unsigned
+
+    $(Sys.WORD_SIZE)-bit unsigned integer type.
+    On $(Sys.WORD_SIZE)-bit systems, `UInt` is an alias for `UInt$(Sys.WORD_SIZE).`
+
+    See also: [`UInt`](@ref)
+    """
+    $(Symbol("UInt", Sys.WORD_SIZE))
+end
+
+
+"""
+    Int <: Signed
+
+`Int` is aliased to `Int64` on 64-bit systems and `Int32` on 32-bit systems.
+
+`Int` is the default integer type in Julia, and should be preferred unless there
+are explicit reasons for using another integer type.
+
+See also: [`$(Symbol("Int", Sys.WORD_SIZE))`](@ref), [`UInt`](@ref).
+
+# Examples
+```jldoctest
+julia> 1 isa Int
+true
+
+julia> typeof(length(Dict())) === Int
+true
+
+julia> Threads.nthreads() isa Int
+true
```

Comment:
These examples are a bit obscure/advanced for a docstring about such a basic, fundamental type (especially the `nthreads()` example. Instead, maybe just:
```suggestion
julia> typeof(length([1,2])) === Int
true
```

Reply:
Good idea not to introduce multi-threading here.